src/Tokenizer.js

```diff
@@ -304,9 +300,9 @@ export class Tokenizer {
       for (i = 0; i < l; i++) {
         this.lexer.state.top = false;
         list.items[i].tokens = this.lexer.blockTokens(list.items[i].text, []);
-        if (list.items[i].tokens.some(t => t.type === 'space')) {
+        if (!list.loose && list.items[i].tokens.some(t => t.type === 'space')) {
           list.loose = true;
-          list.items[i].loose = true;
+          // list.items[i].loose = true;
```

Comment:
Do list items need to be marked as loose or just the list itself? This is a spot we could speed up slightly if we keep it commented out; it doesn't fail any tests.

Reply:
It looks like the parser only uses the `loose` property on the list not the list items. I think it would be fine to not have this on the list items but it might be considered a breaking change.

Reply:
I will leave that up to you then. If needed we can lump it in with other breaking changes at a later date?

Reply:
Actually it looks like that property is not necessarily correct in the list items anyway. Only the list items with a `space` token have the loose property set to true but it should be all items if any of them have a `space` token.

Reply:
Ya I would say leave it right now and we can remove it later.

Reply:
Ok, it's uncommented now.

> Actually it looks like that property is not necessarily correct in the list items anyway

Ah yeah, you're right. I guess that mistake didn't get caught because it's never used anywhere.